src/ros/ros1/core-helper.ts

```diff
@@ -187,7 +187,7 @@ export class XmlRpcApi {
  */
 export class StatusBarItem {
     private item: vscode.StatusBarItem;
-    private timer: NodeJS.Timer;
+    private timer: NodeJS.Timeout;
```

Comment:
Not sure why the type mismatch was not caught by `tsc` but now after switching to `webpack` it becomes a build break.